BabelfishFeatures.cfg

```diff
@@ -1182,6 +1183,7 @@ report_group=DML
 
 [SELECT..UNPIVOT]
 rule=unpivot_clause
+supported-4.6.0=*

```

Comment:
This is not correct since UNPIVOT is not in 5.1.0.  The line `supported-4.6.0=*` means that this is supported in all versions with a higher version number than 4.6.0 (inclusive). And 5.1.0 is higher than 4.6.0. 
The correct config would be `supported-4.6.0-4.*=*` plus a second line `supported-5.2.0=*`

As an example, see `supported-3.5.0-3.*=GEOGRAPHY,GEOMETRY`